bigtable-dataflow-parent/bigtable-beam-import/src/test/java/com/google/cloud/bigtable/beam/it/CloudBigtableBeamIT.java

```diff
@@ -199,67 +175,74 @@ private void testWriteToBigtable() {
             .waitUntilFinish();
 
     Assert.assertEquals(PipelineResult.State.DONE, result);
+
+    try (ResultScanner scanner =
+        connection.getTable(tableName).getScanner(new Scan().setFilter(new KeyOnlyFilter()))) {
+      int count = 0;
+      while (scanner.next() != null) {
+        count++;
+      }
+      Assert.assertEquals(TOTAL_ROW_COUNT, count);
```

Comment:
Is this new functionality? Were we not validating the writes before?

Reply:
no, read & write used to be 2 sequential phases of the same test. This changes them to run in parallel so additional checks are necessary